airbyte-integrations/connectors/source-hubspot/setup.py

```diff
@@ -9,7 +9,6 @@
     "airbyte-cdk",
     "backoff==1.11.1",
     "pendulum==2.1.2",
-    "requests==2.26.0",
```

Comment:
`requests` is already an `airbyte-cdk` transitive dependency.
Pinning it to an "old" 2.26.0 version leads to the following error when running `spec`: 
```
│ │     ┃ Traceback (most recent call last):
│ │     ┃   File "/airbyte/integration_code/main.py", line 8, in <module>
│ │     ┃     from airbyte_cdk.entrypoint import launch
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/__init__.py", line 6, in <module>
│ │     ┃     from .entrypoint import AirbyteEntrypoint
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 17, in <module>
│ │     ┃     import requests
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/requests/__init__.py", line 100, in <module>
│ │     ┃     check_compatibility(urllib3.__version__, chardet_version, charset_normalizer_version)
│ │     ┃   File "/usr/local/lib/python3.9/site-packages/requests/__init__.py", line 85, in check_compatibility
│ │     ┃     raise Exception("You need either charset_normalizer or chardet installed")
│ │     ┃ Exception: You need either charset_normalizer or chardet installed
``` 
Remove `requests` from `setup.py` ensure the connector will use whatever version the CDK needs. The request's CDK version does not lead to this error when it's built inside or python base image.

Reply:
I'm going to fix this here: https://github.com/airbytehq/airbyte/pull/31606

Reply:
@bazarnov let me know when https://github.com/airbytehq/airbyte/pull/31606 is merged and I'll update this branch